### ISSUE TITLE
Presence of an ocular neaplasm

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#10932]
**Preferred term label:**
Presence of an ocular tumor

**Synonyms**
na

**Definition (free text, please give PubMed ID)**
Useful as a parent term to classify the presence of ocular tumors such as (HP:30666).

**Parent term (use hpo.jax.org/app)**
HP:0011793 (Neoplasm by anatomical site)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
NA

**Your nano-attribution (ORCID)**
0000-0001-9042-0508

**Comment by daboe01** [comment]
sorry for the dupe: 100012
